content/js3/blocks/then/index.md

```diff
@@ -0,0 +1,9 @@
++++
+title = 'Making sense of .then()'
```

Comment:
Ditto about missing content

Reply:
oh yes I just set up these blocks based on the headings -- then we reordered so these will all be for week 4